apis/core/bootstrap/shapes/dimension.ts

```diff
@@ -117,7 +118,7 @@ ${shape('dimension/metadata')} {
       ${sh.name} "Description" ;
       ${sh.path} ${schema.description} ;
       ${sh.datatype} ${rdf.langString} ;
-      ${sh.languageIn} ( "en" "de" "fr" "it" ) ;
+      ${sh.languageIn} ${supportedLanguagesRdf} ;
```

Comment:
The additional export is unnecessary. You can [interpolate the array](https://runkit.com/embed/td1qgbq2rpju) directly. Just keep the parens

```suggestion
      ${sh.languageIn} ( ${supportedLanguages} ) ;
```

Reply:
Isn't it missing the quotes around each value?

Reply:
Ah, my bad 😞